scripts/config

```diff
@@ -165,10 +167,4 @@ apply_config() {
 #=================================================
 # GENERIC FINALIZATION
 #=================================================
-# SELECT THE ACTION FOLLOWING THE GIVEN ARGUMENT
-#=================================================
-
-case $1 in
-    show) show_config;;
-    apply) apply_config;;
-esac
+ynh_app_config_run $1
```

Comment:
This isnt enough to adapt the config panel to 1.0 ... the script still contains a lot of `$YNH_CONFIG_MAIN_stuff` and other pieces of code which arent relevant anymore ...

But followup of recent discussions about php-fpm and stuff, considering the current config panel only has "boring" stuff like "overwrite_conf" flags and php-fpm conf stuff which aren't super relevant for this app because it's unlikely to have a super high load, I would just drop the current config panel for this app altogether ...

Reply:
removed